.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
```

Comment:
Would not make solution/alternatives so prominent. Depending on who raises the ticket, they may have ideas or not on how to solve. Would rather go for "Optional: Solution ideas (if you have any)" at the very end